src/msg/async/AsyncMessenger.cc

```diff
@@ -201,6 +201,7 @@ void Processor::accept()
 	accept_error_num = 0;
 	continue;
       } else {
+	--w->references;
```

Comment:
should not decrement refcount if "r == -EINTR".

Reply:
r == -EINTR, w->reference still ++. why not decrement?